frontend/src/styles/index.css

```
@@ -5,15 +5,33 @@
+}
+
+.light {
+  --kepler-background: #f4f7fa;
+  --kepler-surface: #ffffff;
+  --kepler-border-panel: #cbd5e1;
+  --kepler-on-surface: #1e293b;
+  --kepler-on-surface-variant: #475569;
+}
+
 @font-face {
   font-family: "Necosmic";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Necosmic" (font-family-name-quotes)

(font-family-name-quotes)